rhombus/rhombus/scribblings/reference/evt.scrbl

```diff
@@ -73,3 +90,21 @@ count.
  or @rhombus(Evt.never).
 
 }
+
+@doc(
+  interface Synchronizable
+){
+ An interface that a class can implement to make instances of the class usable
+ as an @rhombus(Evt, ~annot).  When a class that implements
+ @rhombus(Synchronizable) is used with @rhombus(Evt.sync) the
+ @rhombus(Synchronizable.to_evt) method is called, and the result is used in
+ the synchronization.
+
+ The interface has a single abstract method:
+
+@itemlist(
+  @item{@rhombus(#,(@rhombus(to_evt, ~datum))()) --- produces an
+  @tech{synchronizable event} that can be used in @rhombus(Evt.sync).}
+)
```

Comment:
Isn't the method `as_evt`?

I also wonder if the fact that the method is called inside a `guard-evt` has any implications that should be documented. (It does definitely seem to me like the right choice!)

Reply:
Good catch on the method name.

I think that saying it gets called during an `Evt.sync` should cover it, but improvements welcome.